RevenueCatUI/CustomerCenter/Data/CustomerCenterActionBridge.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerCenterViewModelPreferenceKey.swift
+//  
+//  Created by Cesar de la Vega on 2024-06-17.
+
+import RevenueCat
+import SwiftUI
+
+/// Helper class that bridges the CustomerCenterActionHandler to the new preference-based system
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@MainActor
+class CustomerCenterActionBridge {
```

Comment:
This bridge is a good idea to never forget™️ calling the deprecated action handlers. However, I'd have it rely on some higher level object instead of on `CustomerCenterAction`, because we cannot add new cases to it without breaking the API. E.g. something in the vein of
```swift
internal enum CustomerCenterInternalAction {
    case `public`(CustomerCenterAction)
    case newCase1
    case newCase2
}
```

Reply:
`legacy` or `deprecated` could work as well, I really like this